common/traits/wc_special_traits.txt

```diff
@@ -405,3 +405,24 @@ secret_patrician = {
 	random = no
 	customizer = no
 }
+# Used for characters "on Draenor"
+on_another_world = {
+	customizer = no
+	pilgrimage = yes
+	cached = yes
+	random = no
+	cannot_marry = yes
+	cannot_inherit = yes
```

Comment:
I would add `incapacitating = yes` because `is_incapable` is very common.
And I would remove `cached = yes` because I doubt it will be needed somewhere.
https://ck2.paradoxwikis.com/Trait_modding

Reply:
I have a feeling that if I add `incapacitating = yes` that it will show the incapable icon on the character instead of the pilgrimage, but I need to check this out first.. So I will return with info when I can.

Reply:
![image](https://user-images.githubusercontent.com/439655/67718529-3d642d80-f9d0-11e9-8d13-2fe128b780de.png)
Seems it does add the incapacitated icon to his char portrait, hence I wanted to avoid using it.

What do you think?

Reply:
I would delete `pilgrimage = yes` then since it's not used in conditions and this icon was the only purpose.

Reply:
But the pilgrimage icon is the only reason this trait really stands out, to be honest.

Giving the `pilgrimage = yes` status makes it all still work as it should plus it gives the aesthetic of them "being away". If we take that away then it becomes less obvious what happened to them once more.